tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_block_config.py

```diff
@@ -0,0 +1,68 @@
+# <snippet name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_block_config.py imports">
+import great_expectations as gx
+from great_expectations.core.batch import BatchRequest
+
+# </snippet>
+
+from great_expectations.core.yaml_handler import YAMLHandler
+
+yaml = YAMLHandler()
+
+# <snippet name="tests/integration/docusaurus/validation/validator/how_to_create_and_edit_expectations_with_instant_feedback_block_config.py get_context">
+context = gx.get_context()
+# </snippet>
+
+datasource_yaml = r"""
```

Comment:
wanting to confirm: @NathanFarmer are we specifically not showing this config portion in the docs?

Reply:
After discussions with @Rachel-Reverie, no I don't think we intend to include this setup. But she can change the snippets while she is writing the `.md` file if she wants.

Reply:
ok sounds good :)